src/js/themes/base.d.ts

```diff
@@ -852,6 +852,7 @@ export interface ThemeType {
     };
     check?: {
       radius?: string;
+      background?: Object;
```

Comment:
Small adjustment for additional type specificity. 

```suggestion
      background?: BackgroundType;
```

Reply:
I had a similar thought at the beginning, but I don't think this suggestion is accurate.
In this case, the background is not treated as a BackgroundType, we are not handling any other background props such as `image`, `repeat` `opacity`, or anything other than the background `color`.
I would agree that the type should be more explicit to the color prop as follows:
```suggestion
      background?: {
          color?: ColorType;
      }
```